vwifi.c

```diff
@@ -97,6 +97,7 @@ struct owl_vif {
         };
         /* Structure for AP mode */
         struct {
+            u8 ap_enable;
```

Comment:
Can you declare it as `bool` instead?
Quote from [Linux kernel coding style](https://www.kernel.org/doc/Documentation/process/coding-style.rst)
> The Linux kernel `bool` type is an alias for the C99 `_Bool` type. bool values can
only evaluate to 0 or 1, and implicit or explicit conversion to `bool`
automatically converts the value to `true` or `false`. When using `bool` types the
!! construction is not needed, which eliminates a class of bugs.

Reply:
Done